docs/ref/files/storage.txt

```diff
@@ -141,6 +150,15 @@ The Storage Class
         the last modified time, this will raise
         ``NotImplementedError`` instead.
 
+    .. method:: move(src_name, dst_name)
+
+        Move the file specified by ``src_name`` to ``dst_name``. If
+        ``dst_name`` already exists ``ValueError`` will be raised. For storage
```

Comment:
since this doc is in the `Storage` class, we should say something like "ValueError should be raised." if that's the expectation of other subclasses.